subsys/bluetooth/services/fast_pair/fmdn/callbacks.c

```diff
@@ -5,6 +5,7 @@
  */
```

Comment:
In the commit message `FMDN callback module`?

Reply:
It's called the FMDN callbacks module, we can rename in a dedicated task but let's use the same naming consistently:

nrf/subsys/bluetooth/services/fast_pair/fmdn/include_priv/fp_fmdn_callbacks.h

Reply:
Sorry, meant `callbacks module`

Reply:
addressed